src/Common/src/Interop/Windows/mincore/Interop.FormatMessage.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        private const int FORMAT_MESSAGE_IGNORE_INSERTS = 0x00000200;
+        private const int FORMAT_MESSAGE_FROM_SYSTEM = 0x00001000;
+        private const int FORMAT_MESSAGE_ARGUMENT_ARRAY = 0x00002000;
+
+        private const int ERROR_INSUFFICIENT_BUFFER = 0x7A;
+
+        [DllImport(Libraries.Localization, CharSet = CharSet.Unicode,  EntryPoint="FormatMessageW", SetLastError = true, BestFitMapping = true)]
+        private static extern int FormatMessage(int dwFlags, IntPtr lpSource_mustBeNull, uint dwMessageId,
+            int dwLanguageId, StringBuilder lpBuffer, int nSize, IntPtr[] arguments);
+
+        /// <summary>
+        ///     Returns a string message for the specified Win32 error code.
+        /// </summary>
+        internal static string GetMessage(int errorCode)
+        {
+            string errorMsg;
+
+            var sb = new StringBuilder(256);
```

Comment:
Since the MaxAllowedBufferSize is in a constant, we should probably have InitialBufferSize in a constant right next to it.  Same goes for the 4 below... should probably be in a BufferSizeIncreaseFactor constant alongsize the other two, since they're all related.